### PR TITLE
feat: add 108th catchup summary

### PR DESCRIPTION
**Pull request by SirusCodes**

### CODE REVIEW

File: summary/sessions/108/content.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ** link:https://twitter.com/SirusTweets[Darshan Rander^] added that how basics things might look there is someone out there who might need help with it.
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] explained why link:https://appwrite.io[Appwrite] has documentation to work with Docker. As it is easier to start with for begineers and Docker is also OCI compilant so if anyone else wants to use it somewhere else they can easily work with image as well.
+* link:https://twitter.com/kaku_jay[Jay Kaku^] shared his experience at a CTF where they had a consistent rank of 8 in first two rounds and could make it to finals because of semester exams.
+* We talked about the link:https://techgenix.com/warp-vs-vpn-online-security[difference between VPN and WARP].
+    ** link:https://1.1.1.1/[WARP] finds a route to the destination server with lesser congestion.
+
+==== Project showcase
+
+* link:https://twitter.com/tusharnankanii[Tushar Nankani^] showcased two of his projects https://tusharnankani.github.io/patternistic[patternistic^] and https://tusharnankani.github.io/resonating-circles[resonating circles^]
```
Comment (tusharnankani) [inline summary/sessions/108/content.adoc:30]:
```suggestion
* link:https://twitter.com/tusharnankanii[Tushar Nankani^] showcased two of his projects experimenting around web animations — 
    ** https://tusharnankani.github.io/patternistic[patternistic^]
    ** https://tusharnankani.github.io/resonating-circles[Resonating Circles^]
```

File: summary/sessions/108/content.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Date: 03-12-2022
+
+Duration: 3 hrs 12 mins
+
+==== Topics Discussed
+
+* We helped Atharva Jadhav with how he can start with backend development.
```
Comment (tusharnankani) [inline summary/sessions/108/content.adoc:7]:
```suggestion
* We helped Atharva Jadhav in figuring out how to begin backend development.
```

File: summary/sessions/108/content.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    ** We suggested him to either start with Flask or Express as a beginner.
+    ** We asked him to learn through YouTube instead of a course as it is free and he might learn something new while looking it up on YouTube instead of following a course.
+    ** link:https://twitter.com/harshgkapadia[Harsh Kapadia^] suggested him https://www.youtube.com/@TraversyMedia[Traversy Media^], https://www.youtube.com/@WebDevSimplified[Web Dev Simplified^] and https://www.youtube.com/@developedbyed[developedbyed^] channels
```
Comment (tusharnankani) [inline summary/sessions/108/content.adoc:10]: `link:` missing

```suggestion
    ** link:https://twitter.com/harshgkapadia[Harsh Kapadia^] suggested a few YouTube Channels —
        *** link:https://www.youtube.com/@TraversyMedia[Traversy Media^]
        *** https://www.youtube.com/@WebDevSimplified[Web Dev Simplified^]
        *** https://www.youtube.com/@developedbyed[developedbyed^] 
```

File: summary/sessions/108/content.adoc [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    ** Yash Pimple suggested https://www.youtube.com/@javascriptmastery[JavaScript Mastery] channel
+* We announced link:https://talks.ourtech.community[OTC Talks^], talks which will happen before catchup once or twice a month.
+    ** You can submit your talk at link:https://catchup.ourtech.community/cfp[catchup.ourtech.community/cfp^]
+* link:https://twitter.com/SirusTweets[Darshan Rander^], link:https://twitter.com/harshgkapadia[Harsh Kapadia^] and link:https://twitter.com/tusharnankanii[Tushar Nankani^] talked about the difference between link:https://stackoverflow.com/questions/25991367/difference-between-throttling-and-debouncing-a-function[throttling and debouce].
+    ** Throttling is when you intentionally delay a call.
+    ** Debouce will merge multiple calls into a single one following some rules.
```
Comment (tusharnankani) [inline summary/sessions/108/content.adoc:16]:
```suggestion
    ** Debouncing will defer the execution of some code after a certain amount of inactivity time.
```

File: summary/sessions/108/content.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] shared his https://github.com/Rishit-dagli/cs-theory-notes[CS theory notes].
+* link:https://twitter.com/PranavDani3[Pranav Dani^] shared about an link:https://www.reddit.com/r/india/comments/n3b9jc/jiofiber_blocked_access_to_rawgithubusercontentcom/[issue^] he is facing with JioFiber, that he couldn't make a request https://raw.githubusercontent.com/ domain.
```
Comment (tusharnankani) [inline summary/sessions/108/content.adoc:18]:
```suggestion
* link:https://twitter.com/PranavDani3[Pranav Dani^] shared about an link:https://www.reddit.com/r/india/comments/n3b9jc/jiofiber_blocked_access_to_rawgithubusercontentcom/[issue^] he is facing with JioFiber, that he couldn't make a request https://raw.githubusercontent.com domain.
```

File: summary/sessions/108/content.adoc [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    ** He fixed it by changing his DNS to 1.1.1.1
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] asked link:https://twitter.com/tusharnankanii[Tushar Nankani^] to write a blog from his tweets about his projects.
```
Comment (tusharnankani) [inline summary/sessions/108/content.adoc:20]:
```suggestion
* link:https://twitter.com/harshgkapadia[Harsh Kapadia^] asked link:https://twitter.com/tusharnankanii[Tushar Nankani^] to write a blog from his tweets about his projects based on Web Animations.
```

File: summary/sessions/108/content.adoc [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+Duration: 3 hrs 12 mins
+
+==== Topics Discussed
+
+* We helped Atharva Jadhav with how he can start with backend development.
+    ** We suggested him to either start with Flask or Express as a beginner.
+    ** We asked him to learn through YouTube instead of a course as it is free and he might learn something new while looking it up on YouTube instead of following a course.
+    ** link:https://twitter.com/harshgkapadia[Harsh Kapadia^] suggested him https://www.youtube.com/@TraversyMedia[Traversy Media^], https://www.youtube.com/@WebDevSimplified[Web Dev Simplified^] and https://www.youtube.com/@developedbyed[developedbyed^] channels
+    ** Yash Pimple suggested https://www.youtube.com/@javascriptmastery[JavaScript Mastery] channel
+* We announced link:https://talks.ourtech.community[OTC Talks^], talks which will happen before catchup once or twice a month.
```
[the remark this answers is not in the document]
Comment (SirusCodes) [reply summary/sessions/108/content.adoc:12]: I feel *or* is better than */*

File: summary/sessions/108/content.adoc [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    ** You can submit your talk at link:https://catchup.ourtech.community/cfp[catchup.ourtech.community/cfp^]
```
[the remark this answers is not in the document]
Comment (SirusCodes) [reply summary/sessions/108/content.adoc:13]: Shit thanks

File: summary/sessions/108/content.adoc (hunk 4 above)
Comment (SirusCodes) [reply summary/sessions/108/content.adoc:16]: That's the implementation thing. Debouncing can also be don't actually call a function till a button is pressed n times

Comment (KartikSoneji) [reply summary/sessions/108/content.adoc:19]: Debouncing is actually a term borrowed from electrical engineering.
https://www.techtarget.com/whatis/definition/debouncing
> Bouncing is the tendency of any two metal contacts in an electronic device to generate multiple signals as the contacts close or open; debouncing is any kind of hardware device or software that ensures that only a single signal will be acted upon for a single opening or closing of a contact.

**Pull request merged**